__tests__/app.test.js

```diff
@@ -103,3 +104,40 @@ describe("GET /api/articles/:article_id", () => {
     expect(msg).toBe(expected);
   });
 });
+describe("GET /api/articles", () => {
+  test("Should return a 200 status", async () => {
+    const { status } = await request(app).get("/api/articles");
+    expect(status).toBe(200);
+  });
+
+  test("Should return an array of articles", async () => {
+    const {
+      body: { article },
+    } = await request(app).get("/api/articles");
+    const expected = {
+      author: "icellusedkars",
+      title: "Eight pug gifs that remind me of mitch",
+      article_id: 3,
+      topic: "mitch",
+      created_at: "2020-11-03T09:12:00.000Z",
+      votes: 0,
+      article_img_url:
+        "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      comment_count: "2",
+    };
+
+    expect(Array.isArray(article)).toBe(true);
+    expect(Array.isArray(article[0])).toBe(false);
+    expect(article[0]).toEqual(expected);
```

Comment:
We are only checking that one article the way we expect. If we are getting 13 articles back from the database we should be checking that they all have the same structure.

Reply:
I've added a new test assertion to loop and check the structure of every article.